src/el/nethermind/nethermind_launcher.star

```diff
@@ -46,7 +46,7 @@ USED_PORTS = {
         ENGINE_RPC_PORT_NUM, shared_utils.TCP_PROTOCOL
     ),
     METRICS_PORT_ID: shared_utils.new_port_spec(
-        METRICS_PORT_NUM, shared_utils.TCP_PROTOCOL
+        METRICS_PORT_NUM, shared_utils.TCP_PROTOCOL, wait=None
```

Comment:
can this be a larger number? does the port never come up?

Reply:
Works fine without wait=None on my laptop, only fails with CI. Also it passes sometimes, fails other times.

Reply:
https://github.com/kurtosis-tech/ethereum-package/pull/467 fixed it.